pypit/arsort.py

```diff
@@ -318,7 +320,7 @@ def sort_write(fitsdict, filesort, space=3):
     # Write
```

Comment:
I propose that we delete the .xml code above (lines 267-301), and it might be better to move the msgs.info message on line 300-301 to line ~322.